www/src/components/Settings.jsx

```diff
@@ -7,7 +7,6 @@ import {
 } from '~paragon-react';
 import { Close } from '~paragon-icons';
 
-import { FEATURES } from '../config';
```

Comment:
eslint warning seen locally. not sure why not caught by CI without further investigation.